pipeline/batch/sql_tasks.py

```diff
@@ -147,6 +148,24 @@ def identified_vendors(metrics_table):
                 LIKE '%squid%'
             OR {metrics_table}.test_keys -> 'received' ->> 3
                 LIKE '%squid%')
+UNION
```

Comment:
Actually no, this is related to privoxy detection. I guess the commit message is indeed a bit misleading.